codegen/projections/white_label/lib/white_label/builders.rb

```diff
@@ -7,118 +7,297 @@
 #
 # WARNING ABOUT GENERATED CODE
 
+require 'stringio'
+
 module WhiteLabel
   # @api private
   module Builders
 
     class CustomAuth
       def self.build(http_req, input:)
+        http_req.http_method = 'POST'
+        http_req.append_path('/')
+        http_req.headers['Content-Type'] = 'application/json'
+        http_req.headers['X-Rpc-Target'] = 'WhiteLabel.CustomAuth'
```

Comment:
Is this header necessary?

Reply:
For this protocol to actually work we need to send the method in some way - it could be through the header or on the url.  I opted for the header as it seemed easier.

Reply:
I thought rpc uses url